.github/workflows/tests.yml

```diff
@@ -16,6 +16,16 @@ jobs:
         php: [7.1, 7.2, 7.3, 7.4]
         laravel: [5.7.*, 5.8.*, ^6.0, ^7.0]
         phpunit: [^7.5, ^8.0]
+        include:
+          - php: 7.2.29
+            laravel: ^7.0
+            phpunit: ^9.0
+          - php: 7.3
+            laravel: ^7.0
+            phpunit: ^9.0
+          - php: 7.4
+            laravel: ^7.0
+            phpunit: ^9.0
```

Comment:
Instead of including these, you'll need to exclude the incompatible PHP versions and PHPUnit 9 versios below in the exclude list.